reactor-core/src/main/java/reactor/core/scheduler/Scheduler.java

```diff
@@ -167,10 +165,28 @@ default Mono<Void> disposeGracefully() {
 	 * <p>The operation is thread-safe but one should avoid using
 	 * start() and dispose() concurrently as it would non-deterministically
 	 * leave the Scheduler in either active or inactive state.
+	 *
+	 * @deprecated Use {@link #init()} instead. The use of this method is discouraged.
+	 * Some implementations allowed restarting a Scheduler, while others did not. One
+	 * of the issues with restarting is that checking
+	 * {@link #isDisposed() the disposed state} is unreliable in concurrent scenarios.
 	 */
+	@Deprecated
 	default void start() {
 	}
 
+	/**
+	 * Instructs this Scheduler to prepare itself for running tasks
+	 * directly or through its {@link Worker}s.
+	 *
+	 * <p><strong>It should be called only once</strong>. Implementations are free to
+	 * throw an exception if this method is called again either during the runtime of the
+	 * scheduler or after it has been disposed via {@link #dispose()}
+	 * or {@link #disposeGracefully()}.
+	 */
+	default void init() {
```

Comment:
could it perhaps delegate to `#start()` in 3.4.x, to ease the transition into the new API ?

Reply:
Thanks for the suggestion - I added this. What do you think about 3.5.x - should the delegation be removed? If it is, it'd not give users much time to migrate. Leaving the delegation for 3.5.x will provide the time to notice the warning about deprecated use of `start()` and to implement a proper `init()`.

Reply:
yeah, I'd say we remove the delegation around 3.6.0 (perhaps even start throwing exceptions at that point? 🤔)